download/archive.md

```diff
@@ -13,30 +13,35 @@ To use our prebuilt, high-performance
 [Docker CE](https://www.docker.com/products/docker-desktop) for your
 platform (Windows, Mac or Linux) and then run the following command:
 
+You can start a container with the following `docker`
+command:
+
 ```bash
-curl -s https://get.fenicsproject.org | bash
+docker run -ti -p 127.0.0.1:8000:8000 -v $(pwd):/root/shared -w /root/shared --name=fenics_legacy ghcr.io/scientificcomputing/fenics-gmsh:2024-05-30
 ```
 
-To run the FEniCS Docker image, use the command `fenicsproject run`. For
-more options and features, see `fenicsproject help`.
-
-Alternatively, you can start a container with the following `docker`
-command:
+and resume this container with
 
 ```bash
-docker run -ti -p 127.0.0.1:8000:8000 -v $(pwd):/home/fenics/shared -w /home/fenics/shared quay.io/fenicsproject/stable:current
+docker container start -i fenics_legacy
 ```
 
-For detailed instructions, see the [FEniCS Reference
+at a later instance.
```

Comment:
I would just remove this and point readers to other sources for help with Docker.